deep_qa/data/tokenizers/word_splitter.py

```diff
@@ -1,17 +1,19 @@
 from collections import OrderedDict
 from typing import List
 
+
 from overrides import overrides
 
-class WordSplitter:
-    """
-    A WordSplitter splits strings into words.  This is typically called a "tokenizer" in NLP, but
-    we need "Tokenizer" to refer to something else, so we're using WordSplitter here instead.
+
+class WordSplitter():
     """
+        A WordSplitter splits strings into words.  This is typically called a "tokenizer" in NLP, but
```

Comment:
Actually, I don't think you made any material changes to this file - you can just revert the whole file.